test/functional/fixtures/reporter/test.js

```diff
@@ -235,6 +235,44 @@ describe('Reporter', () => {
             });
     });
 
+    it('reportTestActionDone should work in the raw tests', () => {
+        const expected = [
+            { expression: 'Selector(\'#button\')', element: { tagName: 'button', attributes: { type: 'button', id: 'button' } } },
+        ];
+
+        function customReporter (log) {
```

Comment:
To simplify code, use the [createReporter](https://github.com/DevExpress/testcafe/blob/master/test/functional/utils/reporter.js) util method here.